### PR TITLE
Update python code to use flatbuffers 2.0 for TRT EP 

### PR DESCRIPTION
**Pull request by chilo-ms**
**Description**: ORT python package doesn't pin flatbuffers version, therefore the latest 2.0 version will be used. This will make our python script which uses the older 1.12 version to fail due to the breaking API change.
After the discussion, we decided to update flatbuffers to 2.0 only on the python side and stick to 1.12 on C++ side.

**Motivation and Context**
- Why is this change required? What problem does it solve? https://github.com/microsoft/onnxruntime/issues/10291
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: onnxruntime/python/tools/quantization/CalTableFlatBuffers/KeyValue.py [hunk 1]
```diff
@@ -35,18 +38,19 @@ def Value(self):
             return self._tab.String(o + self._tab.Pos)
         return None
 
-
+def Start(builder): builder.StartObject(2)
 def KeyValueStart(builder):
-    builder.StartObject(2)
-
-
+    """This method is deprecated. Please switch to Start."""
```
Comment (jywu-msft) [inline onnxruntime/python/tools/quantization/CalTableFlatBuffers/KeyValue.py:43]: sounds like we will need to move away from the deprecated api's ? (KeyValueAddKey, KeyValueAddValue etc.)
if not in this PR, then let's add an item to clean it up later

Comment (chilo-ms) [reply onnxruntime/python/tools/quantization/CalTableFlatBuffers/KeyValue.py:43]: This file is automatically generated by flatbuffers compiler with the schema:
https://github.com/microsoft/onnxruntime-inference-examples/blob/main/quantization/object_detection/trt/yolov3/trt_cal_table.fbs
I'm still figuring out how to generate the include files without deprecated messages.

Comment (jywu-msft) [reply onnxruntime/python/tools/quantization/CalTableFlatBuffers/KeyValue.py:43]: understood. 
I just noticed we are still using the deprecated api's in quant_utils.py is all.

Comment (chilo-ms) [reply onnxruntime/python/tools/quantization/CalTableFlatBuffers/KeyValue.py:43]: Also, from flatbuffers's example code, it has the similar deprecated messages
https://github.com/google/flatbuffers/blob/master/tests/MyGame/Example2/Monster.py#L21

Comment (jywu-msft) [reply onnxruntime/python/tools/quantization/CalTableFlatBuffers/KeyValue.py:43]: yes, it's good they give us heads up on what api's to use/not use.

Comment (chilo-ms) [reply onnxruntime/python/tools/quantization/CalTableFlatBuffers/KeyValue.py:43]: > understood. I just noticed we are still using the deprecated api's in quant_utils.py is all.

Yes, I noticed as well. We should refactor the code in another PR.

**Review by jywu-msft — APPROVED** [review]

**Pull request merged**